Jetsnack/app/build.gradle.kts

```diff
@@ -86,6 +83,14 @@ android {
     }
 }
 
+composeCompiler {
+    // Not required in final Kotlin 2.0.0 release
+    suppressKotlinVersionCompatibilityCheck = "2.0.0-RC1"
```

Comment:
That can be removed if Kotlin 2.0.0-RC2 is used

Reply:
Trying to use `2.0.0-RC2` results in:
```
Compose Multiplatform 1.6.2 doesn't support Kotlin 2.0.0-RC2. Please see https://github.com/JetBrains/compose-jb/blob/master/VERSIONING.md#kotlin-compatibility to know the latest supported version of Kotlin.
```
...where the linked page only references up to `2.0.0-RC1`.  Appreciate this is very much the bleeding edge of compose but it would be nice if the tables could be kept up-to-date with the RC's.

Reply:
We don't control the Jetbrains pages unfortunately. I'm updating this PR now for Kotlin 2.0 though which has been released